### ISSUE TITLE
1.2.7 mangles mapping and m3u

### ISSUE THREAD

**Issue by andonevris** [Threadfin/Threadfin#372]
**Describe the bug**
After upgrading to 1.27 I have encountered some strange behaviour with the channel mapping and threadfin.m3u output file.

I have 2 channels which are the same channel from different sources (using as backup), after the update to 1.2.7 I now have 3 of these channels in the mapping section, a new one appeared at the bottom of the screen separate from the rest of the channels. I can't see where this channel is coming from as the playlist I download from has only 2 instances of this channel.

Furthermore in the threadfin.m3u output I noticed that only 1 of those channels is now listed, the second channel is omitted.  The channels were numbered 1009 and 1010 with 1011 following, in the m3u it lists 1009 and then goes straight to 1011, 1010 is gone.

So in a nutshell I have an extra channel appearing in the mapping list and a missing channel in the m3u list.

I restored a backup and rolled back to the previous version and everything was back to normal so I tried updating to 1.2.7 again to make sure it wasn't a fluke and exactly the same thing happened.

**Server (please complete the following information):**
- OS  windows
- Arch intel x64
- Threadfin Version  1.2.7

**To Reproduce**
1. update to 1.2.7
2. after update a new ghost channel is added to the mapping section (doesn't exist in playlist download) and m3u is missing a channel

**Expected behavior**
show channels correctly

**Comment by Fyb3roptik** [comment]
You need to remove and re-add all filters after this update to fix any duplicates
